src/AppBundle/Entity/Role.php

```diff
@@ -65,7 +66,7 @@ class Role extends AbstractPersonalTranslatable implements TranslatableInterface
      * @var Employee
      *
      * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Employee", inversedBy="roles")
-     * @Type("AppBundle\Entity\Employee")
+     * @Type("array")
```

Comment:
https://github.com/geekhub-php/CheTheatre/blob/master/doc/theatre.json#L480 
this must return object in response too?